### ISSUE TITLE
Some bad results on kid-running case

### ISSUE THREAD

**Issue by STLAND-admin** [google/dynibar#17]
Hi zhengqi！

I tried to train the kid-running case but get some bad results, the only change is the chunk_size and N_rand to fit my GPU memory. 
![image](https://github.com/google/dynibar/assets/62245791/139daf42-ba7e-4132-9142-960289d262df)
So I am also working on the problem and want to know if you can release the Kid-running checkpoint, I didn't find it with those links you offered.

update-------------------------------------------------------------------------------------------------------------
Well, I tried to increase the size of the above two parameters. The results seem much better. However, there are still some artifacts in some view.

![kid-artifact-10](https://github.com/google/dynibar/assets/62245791/b230c788-e04e-410d-9183-6c2b9c96077b)
![kid-artifact-42](https://github.com/google/dynibar/assets/62245791/91662839-ee8c-4ec7-a6b2-a91d33785c43)

Do you have the same problem in your checkpoints or training process? If not, would you please offer the checkpoints?

Thanks!

**Comment by zhengqili** [comment]
Hi. I observed such artifacts before for some of videos I tried. Usually it's due to insufficient initialization and sampling point size. But for this example, using default hyper-parameters is less likely to produce such artifacts. However, I can share the checkpoints I trained if you are interested.

**Comment by STLAND-admin** [comment]
Yes, it will be nice if you can share the checkpoints, my email is styj@connect.hku.hk
